Automattic/crowdsignal-forms
# Remove icons SVG fills

Removing the `fill` property allows for CSS to rule over it, making those icons look more *aligned* with the rest out there.

Note that the Feedback Button icon was changed to use `fill` instead of `stroke`

**Before:**
![fills-before](https://user-images.githubusercontent.com/157240/116139901-f03c4700-a6ac-11eb-8267-81e7378e5fab.gif)

**After:**
![fills-after](https://user-images.githubusercontent.com/157240/116141824-92f5c500-a6af-11eb-8d9a-4078c4fb39e6.gif)

## Test instructions
Checkout and `make client`. Edit a post, type `/crowd` to get all of our blocks on the popup list. Hover the blocks in the list, the icons should get the same color of the text.

Check that Applause and Vote icons preserve functionality and styling on the frontend (Applause allows for a lot of color customization)
